sdk/resource/detect/env.go

```diff
@@ -0,0 +1,72 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package detect
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"os"
+	"strings"
+
+	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/sdk/resource"
+)
+
+// Environment variable used by "env" to decode a resource.
+const envVar = "OTEL_RESOURCE_LABELS"
+
+var (
+	//ErrMissingPairValue happens when a pair does not include value
+	ErrMissingPairValue = errors.New("missing pair value")
```

Comment:
The term pair is not well defined. It would be clear enough to just refer to the value.

```suggestion
	//ErrMissingValue is returned when a resource value is missing.
	ErrMissingValue = errors.New("missing value")
```